src/System.Security.Permissions/src/System.Security.Permissions.csproj

```diff
@@ -176,6 +178,8 @@
     <Compile Include="System\Web\AspNetHostingPermission.cs" />
     <Compile Include="System\Web\AspNetHostingPermissionAttribute.cs" />
     <Compile Include="System\Web\AspNetHostingPermissionLevel.cs" />
+    <Compile Include="System\Xaml\Permissions\XamlAccessLevel.cs" />
+    <Compile Include="System\Xaml\Permissions\XamlLoadPermission.cs" />
   </ItemGroup>
   <ItemGroup Condition="'$(TargetGroup)' == 'netstandard'">
```

Comment:
Below in the section 
```
  <ItemGroup Condition="'$(TargetsNetFx)' == 'true'">
```
Make sure you add a reference to the relevant .NET Framework assemblies (System.Xaml?).

Reply:
Also do a similar thing here: https://github.com/dotnet/corefx/blob/8135319caa7e457ed61053ca1418313b88057b51/src/System.Security.Permissions/ref/System.Security.Permissions.csproj#L12

This is the cause of your build failures.